File: Semestrul 4/Medii de proiectare si programare/Laboratoare/FirmaTransportJava/Model/src/main/java/transport/model/Reservation.java
package transport.model;

public class Reservation extends Entity<Long>{
    private String clientName;
    private Long tripId;
    private int numberOfSeats;

    public Reservation(Long id, String clientName, Long tripId, int numberOfSeats) {
        super(id);
        this.clientName = clientName;
        this.tripId = tripId;
        this.numberOfSeats = numberOfSeats;
    }

    public String getClientName() {
        return clientName;
    }

    public void setClientName(String clientName) {
        this.clientName = clientName;
    }

    public Long getTripId() {
        return tripId;
    }

    public void setTripId(Long tripId) {
        this.tripId = tripId;
    }

    public int getNumberOfSeats() {
        return numberOfSeats;
    }

    public void setNumberOfSeats(int numberOfSeats) {
        this.numberOfSeats = numberOfSeats;
    }
}

File: Semestrul 4/Medii de proiectare si programare/Laboratoare/FirmaTransportC#/Model/Reservation.cs
namespace FirmaTransportC_.Model
{
    public class Reservation : Entity<long>
    {
        private string clientName;
        private long tripId;
        private int numberOfSeats;
        public Reservation(long id, string clientName, long tripId, int numberOfSeats) : base(id)
        {
            this.clientName = clientName;
            this.tripId = tripId;
            this.numberOfSeats = numberOfSeats;
        }
        public string ClientName
        {
            get { return clientName; }
            set { clientName = value; }
        }
        public long TripId
        {
            get { return tripId; }
            set { tripId = value; }
        }

        public int NumberOfSeats
        {
            get { return numberOfSeats; }
            set { numberOfSeats = value; }
        }

        public override string ToString()
        {
            return "Reservation{" +
                "id=" + Id +
                ", clientName=" + clientName +
                ", tripId=" + tripId +
                ", numberOfSeats=" + numberOfSeats +
                '}';
        }
    }
}
